sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/HDFSBackedStateStoreProvider.scala

```diff
@@ -544,6 +597,42 @@ private[sql] class HDFSBackedStateStoreProvider extends StateStoreProvider with
     result
   }
 
+  private def loadMap(startVersion: Long, endVersion: Long): HDFSBackedStateStoreMap = {
+
+    val (result, elapsedMs) = Utils.timeTakenMs {
+      val startVersionMap = Option(loadedMaps.get(startVersion)) match {
+        case Some(value) =>
+          loadedMapCacheHitCount.increment()
+          Option(value)
+        case None =>
+          logWarning(
+            log"The state for version ${MDC(LogKeys.FILE_VERSION, startVersion)} doesn't " +
+              log"exist in loadedMaps. Reading snapshot file and delta files if needed..." +
```

Comment:
nit: don't think this needs to be indented further. cc - @HeartSaVioR to confirm

Reply:
Yeah same indentation may be OK. But it's really a nit.